eessi_bot_event_handler.py

```
@@ -60,9 +69,137 @@ def handle_issue_comment_event(self, event_info, log_file=None):
+            return
+        else:
+            self.log(f"account `{sender}` has permission to send commands to bot")
+
+        # only scan for comments in newly created comments
+        if action == 'created':
+            comment_new = request_body['comment']['body']
+            # next line commented out and replaced to write less to log files
+            # self.log(f"comment created:\n########\n{comment_new.rstrip()}\n########")
+            self.log(f"comment action '{action}' is handled")
+        else:
+            self.log(f"comment action '{action}' not handled")
+            return
+
+        # search for commands in what is new in comment
+        # init comment_update with an empty string or later split would fail if
+        # it is None
+        comment_update = ''
+        commands = []
+        for line in comment_new.split('\n'):
+            # next line commented out to write less to log files
+            # self.log(f"searching line '{line}' for bot command")
+            line_stripped = line.strip()
+            if line.isspace() or len(line_stripped) == 0:
+                # next line commented out to write less to log files
+                # self.log(f"line '{line}' is empty")
+                continue
+            bot_command = get_bot_command(line)
+            if bot_command:
+                try:
+                    ebc = EESSIBotCommand(bot_command)
+                except EESSIBotCommandError as bce:
+                    self.log(f"ERROR: parsing bot command '{bot_command}' failed with {bce.args}")
+                    # next two lines commented out to make the bot less noisy,
+                    # could be enabled again if bot commands accept arg --verbose
+                    # or bot instance has a similar setting
+                    # comment_update += f"\n- parsing bot command `{bot_command}` received"
+                    # comment_update += f" from sender `{sender}` failed"
```

Hmm, I would always do this, mistakes shouldn't be silently ignored?

Maybe this should result in the bot adding an extra comment rather that updating the comment in which the command was found, which is more visible and will trigger a notification to the account that created the comment?

---

Ack. Keeping that type of information in the response to a bot command.